index.js

```diff
@@ -1,24 +1,79 @@
-function rot13(str) { // LBH QVQ VG!
-var shifted = 13;
-var splitted = str.split('');
-var arr = [];
-var inCharCode = splitted.forEach(function(element) {
-arr.push(element.charCodeAt(0));
-return arr;
-});
-var shifterArray = arr.map(function(element) {
-return element += shifted;
-})
-var someString = shifterArray.join( );
-var res = String.fromCharCode(someString);
```

Comment:
fromCharCode expects number as an argument, not a char or a string
Here you could map codes array to chars array using array map function and fromCharCode as callback

Reply:
[65, 66, 67].map((code) => {return String.fromCharCode(code);})